src/move.c

```diff
@@ -1752,22 +1752,6 @@ scrolldown(
     }
 }
 
-/*
- * Return TRUE if scrollup() will scroll by screen line rather than text line.
- */
-    static int
-scrolling_screenlines(int byfold UNUSED)
```

Comment:
It seems this condition is actually only valid in `scrollup()` hence removed the helper function().